### ISSUE TITLE
WFS tests: Schema Validation fails if using local copy of inspire_dls.xsd

### ISSUE THREAD

**Issue by trophies** [Geonovum/etf-test-projects-inspire#73]
The WFS Download service tests appear to do a simple check that asserts that the schemLocation attribute in the GetCapabilities response must point to http://inspire.ec.europa.eu/schemas/inspire_dls/1.0/inspire_dls.xsd

I work on an implementation of the INSPIRE services that stores local copies of the schemas in the web application and references those rather than referencing the schemas on the inspire site.

Is there a another way the schema could be verified rather than asserting it point to the location on the inspire website?  The WMS View service tests don't appear to do this check for the inspire_vs.xsd location but I don't know if they have another way of verifying the schema that is referenced - looks like they just do validation against both the declared schemaLocation and against the schema on the inspire site.

**Comment by thijsbrentjens** [comment]
Good point, thanks. I checked the Technical Guidance just to be sure: there is no requirement that the schemaLocation http://inspire.ec.europa.eu/schemas/inspire_dls/1.0/inspire_dls.xsd shall be used. 

I think your proposal to use the same approach as for WMS View Services will work.
